### PR TITLE
feature(common): adding the file of list opinions in the files [WEBSI…

### PR DESCRIPTION
**Pull request by samquis**
##summary
adding the file of list opinions

### CODE REVIEW

**Comment by olmedo90** [comment]
image

It is not according to Figma. fix the following: (check layer positioning)

**Comment by olmedo90** [comment]
image

It is according to Figma. Continue with the same interest.👍

File: src/App.js [hunk 1]
```diff
@@ -1,9 +1,12 @@
 import Theme from "./Theme/Theme";
 import NavMenu from "./ui/components/navMenu/NavMenu";
+//import Opinions from "./ui/components/opinions/Opiniones"
+import Opinions from "./ui/components/opinions/Listaopinion"
```
Comment (Ditmar) [inline src/App.js]: use simple quotes

File: src/ui/components/opinions/Listaopinion_style.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        padding:'0px',
+        margin:'0px',
+    },
+    sx:{
+
+        display: 'flex',
+        height: '673px',
+        width:'912px',
+        borderRadius:'0.75rem',
+        backgroundColor:'#FFFFFF',
```
Comment (Ditmar) [inline src/ui/components/opinions/Listaopinion_style.js]: the color must have  come from the main Theme file something like this theme.palette.color

File: src/ui/components/opinions/Listaopinion.jsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Grid,Box, Container, Paper } from '@material-ui/core';
+import Filtrars from './filtrars/Filtrar';
+import useStyles from './Listaopinion_style';
+import Opiniones from './Opiniones';
+
+const Listaopiniones = () => {
+  const classes = useStyles();
```
Comment (Ditmar) [inline src/ui/components/opinions/Listaopinion.jsx]: don´t use Spanish use english

**Review by Ditmar — CHANGES_REQUESTED** [review]
please review the comments

**Review by Ditmar — APPROVED** [review]
approved

**Pull request merged**